minari/utils.py

```diff
@@ -292,7 +293,9 @@ def get_average_reference_score(
     for _ in range(num_episodes):
         while True:
             action = policy(obs)
-            obs, _, terminated, truncated, info = env.step(action)
+            obs, _, terminated, truncated, info = env.step(
+                action  # pyright: ignore[reportGeneralTypeIssues]
+            )
```

Comment:
What was the warning here? I cannot see it

Reply:
It was the pyright raising error for reporting General Type Issue while running pre-commit hook.